README.md

```diff
@@ -14,6 +14,23 @@ npm start
 npm run build
 ```
 
+## Deploying
+
+tl;dr:
+```
+npm i -g firebase-tools # only need this once
```

Comment:
Any reason to not have this as a dev dependency?

Reply:
`-g` installs it globally so it’s in the path.